source/enterprise-integration/automation-api.rst

```diff
@@ -269,5 +269,4 @@ name. These fields cannot be used in the POST API.
 
 Datagrid fields appear in human-readable form as tabular data.
 
-More information about Q’s data types can be found in :ref:`Modules,
-Questions, and Documents YAML Reference`.
+More information about Q’s data types can be found in :ref:Modules, Questions, Inputs, and Documents YAML Reference`.
```

Comment:
Should be: `More information about GovReady-Q’s data types`

Reply:
Don't see the difference here.

Reply:
I see it now.  That was a thing from previous documentation.  Fixed though.